main.py

```diff
@@ -79,8 +79,14 @@
                     help='value by which flow will be divided. Original value is 20 but 1 with batchNorm gives good results')
 parser.add_argument('--milestones', default=[100,150,200], metavar='N', nargs='*', help='epochs at which learning rate is divided by 2')
 
+parser.add_argument(
+    "--seed-split",
+    default=None,
+    help="Seed the train-val split to enforce reproducibility (consistent restart too)",
```

Comment:
You can specify the int type here. The default value can still be None, and you won't have to call the `int` function each time you want to use it further